balancer/balancer.go

```diff
@@ -155,6 +155,9 @@ type SubConn interface {
 	// indicate the shutdown operation.  This may be delivered before
 	// in-progress RPCs are complete and the actual connection is closed.
 	Shutdown()
+	// enforceEmbedding is an unexported method to enforce implementers embed
```

Comment:
"...to _force_ implementers _to_ embed..."  or "to enforce _that_ implementers embed".

Reply:
Grammar fixed.